sentinel-dashboard/pom.xml

```diff
@@ -101,7 +101,11 @@
         <dependency>
             <groupId>com.alibaba.csp</groupId>
             <artifactId>sentinel-datasource-nacos</artifactId>
-            <scope>test</scope>
+        </dependency>
```

Comment:
maybe it better to replace the dependence of nacos's spring boot or spring project with  spring-cloud-alibaba?